index.html

```diff
@@ -74,7 +80,7 @@ <h3 class="h">connect:</h3>
 
     <section>
 
-        <h2 class="h">leave a message</h2>
+        <h2 class="h" id="lam">leave a message</h2>
```

Comment:
I noticed you didn't use flexbox or CSS grid to style this section like you did with the experience one. That might be useful to get more experience with the CSS grids and flexbox